tests/dummy/app/pods/area/template.hbs

```diff
@@ -1,4 +1,5 @@
 <div class="dummy-body textarea">
+  <div class="section-header">frost-textarea</div>
```

Comment:
We don't want the section headers in the doc site, so unless there is code to remove these headers please exclude them